src/global/models/selectOption.model.ts

```diff
@@ -0,0 +1,6 @@
+type SelectOption<TValue = number, TLabel = string> = {
+    label: TLabel;
+    value: TValue;
+};
+
+export default SelectOption;
```

Comment:
_**NAB for this PR:**_
We should probably consider exporting this from React CM UI, and replacing the identical one in HC Admin (`church-management`) with the one from React CM UI.

Emphasizing: Not blocking and not recommended right now since we're trying to get this work _DONE!_
Just commenting on things we can improve going forward.

Reply:
That would be cool and nice to have :)